### ISSUE TITLE
No GPS detected: check wiring.

### ISSUE THREAD

**Issue by dnwk** [Xinyuan-LilyGO/LilyGo-LoRa-Series#52]
I am getting this error using the GPS example with this board  https://www.aliexpress.com/item/4001178678568.html

I tried RX PIN 12 or 34. None of them works.

**Comment by dnwk** [comment]
This is TinyGPS example

**Comment by bosstjann** [comment]
I have the same problem. Unable to sort out GPS on v1.1. 
GPS led in blinking but not data on serial1.

**Comment by dykst002** [comment]
Me too

**Comment by bosstjann** [comment]
Hi i have solved my problem with this.
https://github.com/eriktheV-king/TTGO_T-beam_GPS-reset
i needed only version 3.

**Comment by blub2** [comment]
Hi I also have two T22_V1.1 devices (20191212 & 20210222) and the GPS examples are also not working.

I tried with @bosstjann solution with V3 & V2 but the examples are still not working.

I flashed the firmware as well https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/tree/master/firmware
but still no GPS:

> rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0018,len:4
load:0x3fff001c,len:1044
load:0x40078000,len:10124
load:0x40080400,len:5856
entry 0x400806a8
initBoard
SparkFun Ublox Example
No Fix - Num. satellites: 0
No Fix - Num. satellites: 0
No Fix - Num. satellites: 0
No Fix - Num. satellites: 0

**Comment by Siftah** [comment]
> Hi i have solved my problem with this. https://github.com/eriktheV-king/TTGO_T-beam_GPS-reset i needed only version 3.

Thanks, this also resolved my issue! I could see the GPS light flashing red, so I was fairly sure of a fix, but it needed this running to re-enable the NMEA output!.

**Comment by fabiano182** [comment]
Hello everyone,

I have a T22_v1.1 20210222 board and tested all [examples](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/tree/master/examples) of this repo. The picture is from one of them:

![MicrosoftTeams-image](https://user-images.githubusercontent.com/72215944/175660006-1ec93d0d-c3c1-49d7-9390-99ca2b774676.png)

I've also executed the reset ( https://github.com/eriktheV-king/TTGO_T-beam_GPS-reset  ), without success. I checked all pins voltages and everything looks good. I tested in differents days, indoor and outdoor, no one GPS signal was registred and no blinking on LED.

I think my board is broken. What tests I can do to verify?

Best regards
